Sources/VitaminCore/Foundations/Icons/Vitamix.swift

```diff
@@ -1,605 +1,467 @@
-// swiftlint:disable all
-// Generated using SwiftGen — https://github.com/SwiftGen/SwiftGen
 
-import UIKit
+//  Vitamin iOS
```

Comment:
There is a missing `//` on top of the file.
```suggestion
//
//  Vitamin iOS
```